perses/annihilation/relative.py

```diff
@@ -417,6 +417,8 @@ def _determine_core_atoms_in_topology(self, topology, unique_atoms, mapped_atoms
                         hybrid_index = hybrid_map[atom_index]
                         core_atoms.add(hybrid_index)
 
+        assert len(core_atoms) >= 3, 'Cannot run a simulation with fewer than 3 core atoms. System has {len(core_atoms)}'        
```

Comment:
Do we need to be more strict than this? The core atoms also need to be bonded, `1-2-3`.

Reply:
Ummm - yes if there are only three core atoms then they need to be bonded, but I don't think that it's true that it has to be 1-2-3 or 1-3-2 or 1-14-26 (if there are > 3 core atoms). This is just the minimum requirement that makes sure that core_atoms doesn't return empty

Reply:
Totally correct that order doesn't matter as long as they are bonded!
If it's not necessary to check this here, this should be fine!